servers/src/main/java/tachyon/master/file/meta/TTLBucket.java

```diff
@@ -80,4 +82,26 @@ public int compareTo(TTLBucket ttlBucket) {
     }
     return 1;
   }
+
+  /**
+   * Compares to a specific object.
+   *
+   * @param object the object to compare
+   * @return true if object is also {@link TTLBucket} and represents
+   * the same TTLIntervalStartTime
```

Comment:
Can this line be combined to the line above it? It seems that the length hasn't exceeded 100. If this line is kept here, please indent 4 spaces for the continuous line.